src/components/picker/index.tsx

```diff
@@ -218,17 +220,21 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
       <PickerItemsList
         testID={`${testID}.modal`}
         useWheelPicker={useWheelPicker}
+        mode={mode}
+        useDialog={useDialog}
         items={useItems ? items : undefined}
         topBarProps={{
           ...topBarProps,
           onCancel: cancelSelect,
-          onDone: mode === PickerModes.MULTI ? () => onDoneSelecting(multiDraftValue) : undefined
+          onDone: mode === PickerModes.MULTI ? () => onDoneSelecting(multiDraftValue) : undefined,
+          includeStatusBar: useDialog ? false : undefined
```

Comment:
Is the `includeStatusBar` needed for something in private?

Reply:
No, I actually think I comment him the same comment, Ill remove it